src/install/target.rs

```diff
@@ -9,7 +9,4 @@ pub const x86_64: bool = cfg!(target_arch = "x86_64");
 pub const aarch64: bool = cfg!(target_arch = "aarch64");
 
 // Capture if {Wrangler} is in release or debug mode
-pub const DEBUG: bool = cfg!(feature = "debug");
-
-// Capture source location, only in debug mode.
-pub const SOURCE_DIR: &str = env!("SOURCE_DIR");
+pub const DEBUG: bool = cfg!(debug_assertions);
```

Comment:
Actually, the only places this is used is in `wranglerjs::install`: https://github.com/cloudflare/wrangler/blob/19cdb329124a69cb09b7f8c4a5c50b5d85827102/src/wranglerjs/mod.rs#L292-L296
So I wonder if a better check is `current_dir() == env!("CARGO_MANIFEST_DIR")`.